sopel/config/core_section.py

```diff
@@ -1213,6 +1213,68 @@ def homedir(self):
 
     """
 
+    tls_ciphers = ValidatedAttribute(
+        'tls_ciphers',
+        default=":".join(
+            [
+                "ECDHE-ECDSA-AES128-GCM-SHA256",
+                "ECDHE-RSA-AES128-GCM-SHA256",
+                "ECDHE-ECDSA-AES256-GCM-SHA384",
+                "ECDHE-RSA-AES256-GCM-SHA384",
+                "ECDHE-ECDSA-CHACHA20-POLY1305",
+                "ECDHE-RSA-CHACHA20-POLY1305",
+                "DHE-RSA-AES128-GCM-SHA256",
+                "DHE-RSA-AES256-GCM-SHA384",
+            ]
+        )
+    )
+    """The cipher suites enabled for SSL/TLS connections.
+
+    :default: ``ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384``
+
+    The default is the `Mozilla Intermediate configuration`__ as of February
+    2022. This parameter is in OpenSSL format; see the `ciphers manual page`__
+    for your version. This setting cannot be used to configure TLS 1.3.
+
+    .. __: https://wiki.mozilla.org/Security/Server_Side_TLS
+    .. __: https://www.openssl.org/docs/manmaster/man1/ciphers.html
+
+    .. note::
+
+        Cipher selection is also subject to the available SSL/TLS versions,
+        see :attr:`tls_minimum_version`.
+
+    .. code-block:: ini
+
+        tls_ciphers = ECDSA+AES256:!SHA1
+
+    """
+
+    tls_minimum_version = ValidatedAttribute(
+        'tls_minimum_version',
+        default="TLSv1_2",
+    )
+    """The minimum version allowed for SSL/TLS connections.
+
+    :default: ``TLSv1_2``
+
+    You should not set this lower than the default unless the server does not
+    support modern versions. Set this to a `TLSVersion`__ value, e.g. ``TLSv1``
+    or ``TLSv1_3``.
+
+    .. __: https://docs.python.org/3/library/ssl.html#ssl.TLSVersion
```

Comment:
Use Intersphinx refs for this kind of thing.

```suggestion
    You should not set this lower than the default unless the server does not
    support modern versions. Set this to a :class:`~ssl.TLSVersion` value, e.g.
    ``TLSv1`` or ``TLSv1_3``.
```